README.md

```diff
@@ -64,6 +64,25 @@ To mitigate these issues, you can use the `el_client_volume_size` and `cl_client
 
 For optimal performance, we recommend using a cloud provider that allows you to provision Kubernetes clusters with fast persistent storage or self hosting your own Kubernetes cluster with fast persistent storage.
 
+### Shadowforking
+In order to enable shadowfork capabilities, you can use the `network_params.network` flag. The expected value is the name of the network you want to shadowfork followed by `-shadowfork`. For example, to shadowfork the Holesky testnet, you can use the following command:
+```yaml
+...
```

Comment:
Should we tell people that `persistent` is required? Or does that get auto enabled?

Reply:
if persistent and node selectors are requirements for k8s; lets also mention that for now its limited to single host k8s

Reply:
Done